web/api/netdata-swagger.json

```diff
@@ -1224,12 +1242,158 @@
+      ]
+    },
+    "data_array": {
+      "description": "Data response in array format",
+      "allOf": [
+        {
+          "$ref": "#/definitions/data"
+        },
+        {
+          "properties": {
+            "result": {
+              "type": "array",
+              "items": {
+                "type": "number"
+              }
+            }
+          }
+        }
+      ]
+    },
+    "data_csvjsonarray": {
+      "description": "Data response in csvjsonarray format",
+      "allOf": [
+        {
+          "$ref": "#/definitions/data"
+        },
+        {
+          "properties": {
+            "result": {
+              "description": "The first inner array contains strings showing the labels of each column, each subsequent array contains the values for each point in time",
+              "type": "array",
+              "items": {
+                "type": "array",
+                "items": {}
+              }
+            }
+          }
+        }
+      ]
+    },
+    "data_datatable": {
+      "description": "Data response in datatable / datasource formats (suitable for Google Charts)",
+      "allOf": [
+        {
+          "$ref": "#/definitions/data"
+        },
+        {
+          "properties": {
+            "result": {
+              "type": "object",
+              "properties": {
+                "cols": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "description": "Always empty - for future use"
+                      },
+                      "label": {
+                        "description": "The dimension returned from the chart"
+                      },
+                      "pattern": {
+                        "description": "Always empty - for future use"
+                      },
+                      "type": {
+                        "description": "The type of data in the column / chart-dimension"
+                      },
+                      "p": {
+                        "description": "Contains any annotations for the column"
+                      }
+                    },
+                    "required": [
+                      "id",
+                      "label",
+                      "pattern",
+                      "type"
+                    ]
+                  }
+                },
+                "rows": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "c": {
+                        "type": "array",
+                        "items": {
+                          "properties": {
+                            "v": {
+                              "description": "Each value in the row is represented by a c value with five v fields: data, null, null, 0, the value. This format is fixed by the Google Charts API."
```

Comment:
I understood the fact that you are talking about the keys when you wrote "represented by a c value", but is it not better to give some emphasis for these variables to give the notion that is not only a consonant?

Reply:
Good point, updated.